lib/modules/manager/terraform/modules.ts

```diff
@@ -64,7 +64,10 @@ export function analyseTerraformModule(dep: PackageDependency): void {
     dep.datasource = BitBucketTagsDatasource.id;
   } else if (gitTagsRefMatch?.groups) {
     dep.depType = 'module';
-    if (gitTagsRefMatch.groups.path.includes('//')) {
+    if (azureDevOpsSshRefMatch?.groups) {
+      dep.depName = `${azureDevOpsSshRefMatch.groups.organization}/${azureDevOpsSshRefMatch.groups.project}/${azureDevOpsSshRefMatch.groups.repository}${azureDevOpsSshRefMatch.groups.modulepath}`;
+      dep.packageName = azureDevOpsSshRefMatch.groups.url;
+    } else if (gitTagsRefMatch.groups.path.includes('//')) {
```

Comment:
In this case we are defaulting to `dep.currentValue = gitTagsRefMatch.groups.tag` as per:
https://github.com/renovatebot/renovate/blob/b34c58c3ee2e094c4d5b96f51054f04e919451d7/lib/modules/manager/terraform/modules.ts#L79
are we ok with that as opposed to  `dep.currentValue = azureDevOpsSshRefMatch.groups.tag;`?

Reply:
they both catch the same tag, i actually reviewed the code with maron on this

Reply:
than wont `dep.currentValue = azureDevOpsSshRefMatch.groups.tag;` in the given block would be cleaner and more future proof? 😅

Reply:
ok I changed it just in case we have different cases.

Reply:
azure works with git tags, so it just has a different domain/host
tags will never be different i believe, 
but if it's clearer to the eyes we can do that